packages/api-docs-builder-core/baseUi/projectSettings.ts

```diff
@@ -27,6 +27,9 @@ export const projectSettings: ProjectSettings = {
   getHookInfo: getBaseUiHookInfo,
   translationLanguages: LANGUAGES,
   skipComponent: () => false,
+  skipHook: (filename) => {
+    return filename.match(/(useSlotProps)/) !== null;
```

Comment:
This hook was anyway not documented, so I've added this option to skip, otherwise the docgen generation failed. It's an optional prop it won't break other ts projects.